pages/verify-identity.tsx

```diff
@@ -96,74 +88,107 @@ export default function IdentityVerificationForm() {
-                  </PopoverTrigger>
-                  <PopoverContent>
-                    <PopoverBody>
-                      <Image src="/assets/wallet-card-example.png" height={200} width={320} />
-                    </PopoverBody>
-                    <PopoverFooter backgroundColor="background.grayHover">
-                      <Text textStyle="caption">
-                        {`Locate your User ID on the Wallet Card that came with your Permit.`}
-                      </Text>
-                    </PopoverFooter>
-                  </PopoverContent>
-                </Popover>
-                <FormHelperText>
-                  {`You can find your user ID on the back of your wallet card. If you cannot find your
-              wallet card, please call RCD at 604-232-2404.`}
-                </FormHelperText>
-              </Flex>
-            </FormControl>
-            <FormControl isRequired textAlign="left" marginBottom="48px">
-              <FormLabel>{`Last 4 digits of your phone number`}</FormLabel>
-              <Input
-                width="184px"
-                value={phoneNumberSuffix}
-                onChange={event => setPhoneNumberSuffix(event.target.value)}
-              />
-            </FormControl>
-            <FormControl isRequired textAlign="left" marginBottom="20px">
-              <FormLabel>{`Date of Birth`}</FormLabel>
-              <Input
-                type="date"
-                width="184px"
-                value={dateOfBirth}
-                onChange={event => setDateOfBirth(event.target.value)}
-              />
-              <FormHelperText>
-                {`Please enter your date of birth in YYYY-MM-DD format. For example, if you were born on
-            20th August 1950, you would enter 1950-08-20`}
-              </FormHelperText>
-            </FormControl>
-            <Flex width="100%" justifyContent="flex-end">
-              <Link href="/">
-                <Button variant="outline" marginRight="12px">{`Go back to home page`}</Button>
-              </Link>
-              <Button
-                variant="solid"
-                disabled={loading || !userId || !phoneNumberSuffix || !dateOfBirth}
-                loading={loading}
-                onClick={handleSubmit}
-              >{`Continue`}</Button>
-            </Flex>
+            <Formik
+              initialValues={{
+                userId: 0,
+                phoneNumberSuffix: '',
+                dateOfBirth: '',
+              }}
+              validationSchema={verifyIdentitySchema}
+              onSubmit={handleSubmit}
+            >
+              {({ values }) => (
+                <Form style={{ width: '100%' }} noValidate>
+                  <Box marginBottom="48px" textAlign={'left'}>
+                    <NumberField
+                      name="userId"
+                      label="User ID number"
+                      required={true}
```

Comment:
For future reference, for boolean props, you can just write the following to set it to `true`
```suggestion
                      required
```